### PR TITLE
[BananaMecânica] added website support

### PR DESCRIPTION
**Pull request by M4lho**

### CODE REVIEW

File: src/web/mjs/connectors/BananaMecanica.mjs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import WordPressMadaraNovel from './templates/WordPressMadaraNovel.mjs';
+
+export default class BananaMecanica extends WordPressMadaraNovel {
+
+    constructor() {
+        super();
+        super.id = 'bananamecanica';
+        super.label = 'Banana Mecânica';
+        this.tags = [ 'manga', 'webtoon', 'portuguese', 'scanlation' ];
```
Comment (27136773) [inline src/web/mjs/connectors/BananaMecanica.mjs:9]: Are there `manga` on the website? Seems to me like only `webtoon` is present.
Is this really a `scanlation` group or just another aggregator?

File: src/web/mjs/connectors/BananaMecanica.mjs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import WordPressMadaraNovel from './templates/WordPressMadaraNovel.mjs';
+
+export default class BananaMecanica extends WordPressMadaraNovel {
```
Comment (27136773) [inline src/web/mjs/connectors/BananaMecanica.mjs:3]: Are there any novels on the website? If not just use `WordPressMadara`

Comment (M4lho) [reply src/web/mjs/connectors/BananaMecanica.mjs:3]: This one: https://leitorbm.com/manga/a-noiva-do-mal/

File: src/web/mjs/connectors/BananaMecanica.mjs (hunk 1 above)
Comment (M4lho) [reply src/web/mjs/connectors/BananaMecanica.mjs:9]: Is a Scanlation group of Yaoi, link of a [banner group](https://leitorbm.com/wp-content/uploads/WP-manga/data/manga_5fc7c46f26ef9/e28146e25cd5d4af2fd6fc2c687d108e/00-capa15-e-1_01.jpg), a [manga](https://leitorbm.com/manga/usagi-no-mori/).

**Review by 27136773 — APPROVED** [review]
Looks good

**Pull request merged**